src/main/java/se/sundsvall/contract/integration/db/model/converter/LeaseFeesConverter.java

```diff
@@ -4,12 +4,14 @@
 import static java.util.function.Predicate.not;
 
 import jakarta.persistence.AttributeConverter;
+import jakarta.persistence.Converter;
 import jakarta.persistence.PersistenceException;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
 
 import se.sundsvall.contract.model.LeaseFees;
 
+@Converter
```

Comment:
This isn't really needed, since the converter is explicitly set using `@Convert(converter = LeaseFeesConverter.class)` in `LandLeaseContracyEntity`

Reply:
Ah, IntelliJ was complaining, I'll remove it :)